shell/i18n/fi.i18n.json

```diff
@@ -1685,12 +1685,12 @@
           "subtext": "Käyttäjät joilla on tämän domainin sähköpostiosoite tulevat olemaan tämän palvelimen organisaation jäseniä."
         },
         "gappsToggle": {
-          "label": "G Suiten kautta kirjautuneet käyttäjät",
+          "label": "Google Workspace kautta kirjautuneet käyttäjät",
```

Comment:
Actually, while we have @xet7 here.... the text in the setup wizard and the admin panel on the English version are identical, see above in the PR, but they differ slightly in the translation. Should they be the same in the Finnish version?

Reply:
Yes, when it's ` .... kautta` it's like `via Google Workplace` and requires it to be `Google Workspacen`

Reply:
I mean, shouldn't line 713 and line 1688 be the same? And shouldn't line 718 and 1693 be the same?

In the English version, line 759 and 1739 are identical, and line 764 and 1744 are identical.

Reply:
@ocdtrekkie 

Yes, everywhere should be `G Suiten kautta => Google Workplacen kautta`

Other language use word `via` or similar in that language. Finnish uses `n`